samples/demo/src/main.ts

```diff
@@ -1,3 +1,24 @@
+const { useAzureMonitor, AzureMonitorOpenTelemetryOptions } = require("@azure/monitor-opentelemetry");
+const { Resource } = require("@opentelemetry/resources");
+const { SemanticResourceAttributes } = require("@opentelemetry/semantic-conventions");
+
+const customResource = new Resource({
+    [SemanticResourceAttributes.SERVICE_NAME]: "radius-demo-application",
+    [SemanticResourceAttributes.SERVICE_NAMESPACE]: "radius-demo-application-ns",
+    [SemanticResourceAttributes.SERVICE_INSTANCE_ID]: "radius-demo-application-node2",
```

Comment:
This is a big part of the reason why I wanted to use the collector based approach 😆 - this puts a bunch of configuration into the code. 

I'll revisit this when I do the prototyping on collectors.

Likewise I want to make sure this is generic and not specific to Azure monitor. This was a good approach to get the prototyping part of the app done, but I want to see if we can do better 👍

Reply:
Yup, this is scoped to our defined demo/mvp scenario. The concept of a connection string can be generalized as we work through the details.